lib/livereload.js

```diff
@@ -13,9 +13,9 @@ var fs = require('fs');
 var url = require('url');
 var logger = require('./utils/logger');
 var helpers = require('./utils/helpers');
+var _ = require('lodash');
```

Comment:
Consider referencing https://www.npmjs.com/package/lodash.merge instead of all of lodash which is rather big.